src/App.tsx

```diff
@@ -1,24 +1,220 @@
-/* eslint-disable max-len */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable react-hooks/rules-of-hooks */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { Loader } from './components/Loader';
+import { FilterTypes } from './components/TodoFilter';
+import { Todos } from './components/Todos';
+import { Todo } from './types/Todo';
+import { Footer } from './components/Footer';
+import {
+  deleteTodos, getTodos, patchTodos, postTodos,
+} from './api/todos';
 
-const USER_ID = 0;
+export const USER_ID = '10682';
+
+export enum TodoErros {
+  Add = ' Unable to add a todo',
+  Delete = 'Unable to delete a todo',
+  Update = 'Unable to update a todo',
+  ErrorTodo = 'Can not find todos',
+}
 
 export const App: React.FC = () => {
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [input, setInput] = useState<string>('');
```

Comment:
Move the if statement after the hooks. Hooks should generally be called in the same order on every render to ensure consistent behavior. This means that hooks should be called at the top level of your functional component, and they should not be called conditionally or inside loops.